packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/InputEditor.js

```diff
@@ -6,35 +6,25 @@ import ContentEditable from 'dmn-js-shared/lib/components/ContentEditable';
 import Input from 'dmn-js-shared/lib/components/Input';
 import InputSelect from 'dmn-js-shared/lib/components/InputSelect';
 
-
 export default class InputEditor extends Component {
 
   constructor(props, context) {
     super(props, context);
 
+    const { defaultExpressionLanguage } = this.props;
+
     this.setExpressionLanguage = (expressionLanguage) => {
       this.handleChange({ expressionLanguage });
     };
 
-    this.makeScript = (event) => {
-      event.preventDefault();
-      event.stopPropagation();
-
-      this.setExpressionLanguage('FEEL');
-    };
```

Comment:
To simplify this a bit here, I've discarded the one-line input field and would show the multi-line editor every time.
As configuration user, I always find it annoying being forced to do that extra click for complex expressions and for simple ones I don't mind having a bit of empty space below either way.

Can of course revert this part, as it is not really necessary for making the expression languages configurable.